presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -355,11 +356,13 @@ public void testTableNameAmbiguity()
         session.execute("CREATE TABLE keyspace_4.\"TaBlE_4\" (column_4 bigint PRIMARY KEY)");
         session.execute("CREATE TABLE keyspace_4.\"tAbLe_4\" (column_4 bigint PRIMARY KEY)");
 
+        Thread.sleep(5000);
```

Comment:
Why do we need the sleep?

Reply:
What metadata is being refreshed asynchronously?  Can you help me explain why the changes in this PR require this new sleep?

Reply:
Without this delay, the test might run into race conditions where it tries to access or query the newly created schema elements before they are fully registered in the system, leading to test failures.The driver now performs more rigorous checks to ensure schema agreement across nodes before refreshing the metadata. This could introduce a delay, especially if schema changes are not immediately visible across the cluster, potentially causing issues like the one we are  facing where the test fails if you don't include a Thread.sleep. The driver waits for all nodes to agree on the schema before completing the operation, which wasn't as strictly enforced in earlier versions.

Reply:
Doesn't the `assertContainsEventually` call immediately after this sleep handle this race condition?

Reply:
No , without the Thread.sleep its failing and after adding the sleep it passes

Reply:
OK.  Please add a comment explaining why we need the sleep.

Reply:
@tdcmeehan I have added the comment . Thanks